src/components/MapDetail/InfoWithIcon.tsx

```diff
@@ -0,0 +1,95 @@
+import styled, { css } from 'styled-components';
+import Booth from '../../types/Booth';
+import BoothDay from '../../types/BoothDay';
+
+const MapInfoMiddle = styled.div<{$small: string }>`
+        padding: 1rem;
+        div{
+            display: flex;
+            align-items: center;
+            letter-spacing: -0.04348rem;
+        }
+
+        ${(props) => (props.$small === 'false') && css`
+            div{
+                padding: 1rem;
+            }
+            svg{
+            margin: 0 1.5rem;
+            }
+        `}
+
+        ${(props) => (props.$small === 'true') && css`
+            font-size: 12px;
+            svg{
+                width: 13px;
+                margin-right: 1.2rem;
+            }
+            p{
+                letter-spacing: 0.01rem;
+            }
+        `}
+    `;
+
+type InfoWithIconProps = {
+  small: 'true' | 'false';
+  booth: Booth;
+  boothDay: BoothDay;
+}
+
+export default function InfoWithIcon({
+  small,
+  booth,
+  boothDay,
+}: InfoWithIconProps) {
+  if (!booth || !boothDay) { return []; }
```

Comment:
빈배열을 리턴하는 것보다 null을 리턴해주는게 좋을 것 같아요~
```suggestion
  if (!booth || !boothDay) { 
    return null;
}
```